PurchasesTests/Misc/SystemInfoTests.swift

```diff
@@ -4,71 +4,66 @@ import Nimble
+        expect(SystemInfo.proxyURL).to(beNil())
 
         let url = URL(string: "https://my_url")
-        RCSystemInfo.setProxyURL(url)
+        SystemInfo.proxyURL = url
 
-        expect(RCSystemInfo.serverHostURL()) == url
-        expect(RCSystemInfo.proxyURL()) == url
+        expect(SystemInfo.serverHostURL) == url
+        expect(SystemInfo.proxyURL) == url
 
-        RCSystemInfo.setProxyURL(nil)
-        expect(RCSystemInfo.proxyURL()).to(beNil())
+        SystemInfo.proxyURL = nil
+        expect(SystemInfo.proxyURL).to(beNil())
 
-        expect(RCSystemInfo.serverHostURL()) == defaultURL
+        expect(SystemInfo.serverHostURL) == defaultURL
     }
 
-    func testSystemVersion() {
-        expect(RCSystemInfo.systemVersion()) == ProcessInfo().operatingSystemVersionString
+    func testsystemVersion() {
+        expect(SystemInfo.systemVersion) == ProcessInfo().operatingSystemVersionString
     }
 
     func testPlatformFlavor() {
         let flavor = "flavor"
-        let systemInfo = RCSystemInfo(platformFlavor: flavor,
-                                      platformFlavorVersion: "foo",
-                                      finishTransactions: false)
+        let systemInfo = try! SystemInfo(platformFlavor: flavor,
+                                           platformFlavorVersion: "foo",
+                                           finishTransactions: false)
         expect(systemInfo.platformFlavor) == flavor
     }
 
     func testPlatformFlavorVersion() {
         let flavorVersion = "flavorVersion"
-        let systemInfo = RCSystemInfo(platformFlavor: "foo",
-                                      platformFlavorVersion: flavorVersion,
-                                      finishTransactions: false)
+        let systemInfo = try! SystemInfo(platformFlavor: "foo",
+                                           platformFlavorVersion: flavorVersion,
+                                           finishTransactions: false)
         expect(systemInfo.platformFlavorVersion) == flavorVersion
     }
 
     func testPlatformFlavorAndPlatformFlavorVersionMustSimultaneouslyExistOrNotExist() {
-        expectToNotThrowException {
-            _ = RCSystemInfo(platformFlavor: "a", platformFlavorVersion: "b", finishTransactions: true)
-        }
-        expectToThrowException(.parameterAssert) {
-            _ = RCSystemInfo(platformFlavor: nil, platformFlavorVersion: "b", finishTransactions: true)
-        }
-        expectToThrowException(.parameterAssert) {
-            _ = RCSystemInfo(platformFlavor: "a", platformFlavorVersion: nil, finishTransactions: true)
-        }
-            
-        expectToNotThrowException {
-            _ = RCSystemInfo(platformFlavor: nil, platformFlavorVersion: nil, finishTransactions: true)
-        }
+        expect(try SystemInfo(platformFlavor: "a", platformFlavorVersion: "b", finishTransactions: true)).toNot(throwError(SystemInfo.SystemInfoError.invalidInitializationData))
```

Comment:
awesome! 
although I honestly feel like breaking line here might make it easier to read

Reply:
Totally agree, updating.